lib/scala/kg/src/main/scala/stores/kg/mem/MemKgStore.scala

```diff
@@ -125,9 +125,9 @@ class MemKgStore extends KgCommandStore with KgQueryStore {
     )
   }
 
-  final override def getMatchingNodes(limit: Int, offset: Int, query: KgNodeQuery): List[KgNode] = {
+  final override def getMatchingNodes(limit: Int, offset: Int, query: KgNodeQuery, sorts: Option[List[KgNodeSort]]): List[KgNode] = {
     val results: PagedResults[SearchResult] = lucene.query().filter(toSearchTerms(query):_*).limit(limit).offset(offset).search()
-    results.results.toList.map(searchResult => nodesById(searchResult(LuceneFields.nodeId)))
+    KgNodeSort.sort(results.results.toList.map(searchResult => nodesById(searchResult(LuceneFields.nodeId))), sorts.getOrElse(List()))
```

Comment:
As I said above, move the .sort method to this class.

Reply:
I've realized that I'm sorting the results of the query after offset and limit were performed, so I am looking into using the lucene query sort